### DISCUSSION TITLE
This add-on is not compatible...

### DISCUSSION THREAD

**Discussion by pin010** [hassio-addons/addon-vscode#233] [Q&A] [answered]
Hi, I can't install the add-on on my raspberry PI3-B V1.2 Home Assistant OS core-2021.4.3 32bit Linux 5.4.83-v7
Visual Studio Code Version: 1.55.1 on a pc in the same network OS: Windows_NT x64 10.0.19042
The error I get is:
<b> "This add-on is not compatible with the processor of the device or with the operating system installed on the device." </b>
Do you think that by installing the 64bit version this would be possible?
Thanks so much

**Comment by frenck** [comment, marked as answer]
This add-on needs a 64 bit CPU & Operating System.

Considering a Raspberry Pi 3, yes, you could meet the requirements, install and use the add-on. But since you only have 1Gb of memory available: I would strongly advise against it.

The pi3 is simply not powerful enough.

**Reply by pin010** [reply]
Quick answer. Thank you.
